src/styles/card.scss

```diff
@@ -0,0 +1,75 @@
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 200px));
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  border: $gray-border;
+  padding: 32px 16px 16px;
+  border-radius: $rounding;
+  transition: $transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    margin-bottom: 40px;
+    padding-inline: 3px;
+    min-height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover .card__title {
+    color: $title-hover;
```

Comment:
Ensure that the SCSS variable `$title-hover` is defined in your stylesheets. If it's not defined, it will cause an error during the SCSS compilation.